src/webhook/validation/dynakube/validation.go

```diff
@@ -22,26 +22,24 @@ type dynakubeValidator struct {
 	cfg       *rest.Config
 }
 
-func newDynakubeValidator(apiReader client.Reader, cfg *rest.Config) admission.Handler {
+var _ admission.Handler = &dynakubeValidator{}
+
+func newDynakubeValidator(apiReader client.Reader, cfg *rest.Config, clt client.Client) admission.Handler {
```

Comment:
```suggestion
func newDynakubeValidator(clt client.Client, apiReader client.Reader, cfg *rest.Config) admission.Handler {
```

https://github.com/Dynatrace/dynatrace-operator/blob/main/doc/coding-style-guide.md#function-parameter-and-return-value-order